lib/src/formatter.dart

```diff
@@ -173,9 +173,7 @@ class SimpleFormatter implements ReportFormatter {
     var tableWidth = max(_summaryLength, longest + largestCountGuess);
     var pad = tableWidth - longest;
     var line = ''.padLeft(tableWidth, '-');
-    out.writeln(line);
-    out.writeln('Counts');
-    out.writeln(line);
+    out..writeln(line)..writeln('Counts')..writeln(line);
```

Comment:
Your call, but I find this harder to read (and disagree with the lint in general). Having each line of output on a different line in the code makes it more clear for me.